python/src/subtensornodeapi/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
samething, let's centralized where we have the VERSION. And change the update_version script aligned with that